packages/datadog-plugin-fastify/test/code_origin.spec.js

```diff
@@ -4,16 +4,13 @@ const axios = require('axios')
 const semver = require('semver')
 const agent = require('../../dd-trace/test/plugins/agent')
 const { getNextLineNumber } = require('../../dd-trace/test/plugins/helpers')
-const { NODE_MAJOR } = require('../../../version')
 
 describe('Plugin', () => {
   let fastify
   let app
 
   describe('fastify', () => {
     withVersions('fastify', 'fastify', (version, _, specificVersion) => {
-      if (NODE_MAJOR <= 18 && semver.satisfies(specificVersion, '>=5')) return
```

Comment:
aren't we still supporting node18 in v5? I might've misunderstood but I thought we wanted `master` to look as closely as possible as release branches so as to avoid merge conflicts

Reply:
Yes, I guess that should stay until we stop supporting v5